scripts/dts/extract_dts_includes.py

```diff
@@ -196,6 +196,18 @@ def find_parent_irq_node(node_address):
 
     return reduced[phandles[interrupt_parent]]
 
+def find_parent_prop(node_address, prop):
+    address = ''
+
+    for comp in node_address.split('/')[1:-1]:
+        address += '/' + comp
+
+    if prop in reduced[address]['props']:
+        parent_prop = reduced[address]['props'].get(prop)
+    else:
+        raise("Parent doesn't have requested property")
```

Comment:
will this error provide enough context information to know what to fix?  Which node?  What property?

Reply:
Right, could be enhanced easily.